Mathlib/RepresentationTheory/GroupCohomology/Hilbert90.lean

```diff
@@ -71,37 +72,36 @@ section
 open Hilbert90
 variable {K L : Type*} [Field K] [Field L] [Algebra K L] [FiniteDimensional K L]
 
-theorem hilbert90 (f : (L ≃ₐ[K] L) → Lˣ)
-    (hf : ∀ (g h : (L ≃ₐ[K] L)), f (g * h) = g (f h) * f g) :
-    ∃ β : Lˣ, ∀ g : (L ≃ₐ[K] L), f g * Units.map g β = β := by
+/-- Given a finite extension of fields and a function `f : Aut_K(L) → Lˣ` satisfying
+`f(gh) = g(f(h)) * f(g)` for all `g, h : Aut_K(L)`, there exists `β : Lˣ` such that
+`g(β)/β = f(g)` for all `g : Aut_K(L).` -/
+theorem hilbert90 (f : (L ≃ₐ[K] L) → Lˣ) (hf : IsMulOneCocycle f) :
```

Comment:
I think we usually give it a name like `isMulOneCoboundary_of_isMulOneCocycle_of_aut_to_units`